packages/genesys-spark-tokens/data/ui/forms_and_inputs.json

```diff
@@ -1399,6 +1399,12 @@
         "value": "{theme.border_radius.interactive.small}",
         "type": "borderRadius"
       }
+    },
+    "set": {
+      "height": {
+        "value": "{interactive.xs.size}",
+        "type": "sizing"
+      }
```

Comment:
This is the substantive change in theis PR.

The other changes are related to the property rename from `slider` -> `range_slider` here.